roles/kubernetes/node/templates/kubelet-config.v1beta1.yaml.j2

```diff
@@ -23,7 +23,7 @@ enforceNodeAllocatable:
 {% endfor %}
 {% endif %}
 staticPodPath: {{ kube_manifest_dir }}
-cgroupDriver: {{ kubelet_cgroup_driver }}
+cgroupDriver: {{ kubelet_cgroup_driver | default('systemd') }}
```

Comment:
Is there a corner case where `kubelet_cgroup_driver` is undefined? This value is computed in `roles/kubernetes/node/tasks/facts.yml` and should never be undefined.

Reply:
> Is there a corner case where `kubelet_cgroup_driver` is undefined? This value is computed in `roles/kubernetes/node/tasks/facts.yml` and should never be undefined.

Run ansible-playbook with `--tags=kubelet,kubeadm`.

Inspired by <https://github.com/kubernetes-sigs/kubespray/blob/02b6e4833a03be2080932bf97d08a30d3d820726/roles/container-engine/cri-o/defaults/main.yml#L3>.

Reply:
> Run ansible-playbook with --tags=kubelet,kubeadm.

That is an exceptional corner case indeed.

Setting a default value for the `kubelet_cgroup_driver` would break the current detection logic so it does seem like the less clean approach is the more viable one.